lib/python/rose/macro.py

```diff
@@ -48,7 +48,9 @@
 ERROR_LOAD_CONFIG_DIR = "{0}: not an application or suite directory.\n"
 ERROR_LOAD_MACRO = "Could not load macro {0}: {1}"
 ERROR_LOAD_METADATA = "Could not load metadata {0}\n"
-ERROR_LOAD_CHOSEN_META_PATH = "Could not find metadata for {0}, using {1}\n"
+ERROR_LOAD_CHOSEN_META_PATH = (
+    "Could not find metadata for {0}, using {1}\n"
+    "To suppress these warnings run 'rose edit --no-warn version")
```

Comment:
Sorry, just ran a final walk through and noticed the missing `'` after `version`.